tools/mo/openvino/tools/mo/utils/get_ov_update_message.py

```diff
@@ -14,3 +14,12 @@ def get_ov_update_message():
     link = 'https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html?cid=other&source=prod&campid=ww_2022_bu_IOTG_OpenVINO-2022-1&content=upg_all&medium=organic'
 
     return msg_fmt.format(link) if current_date >= expected_update_date else None
+
+
+def get_ov_api20_message():
+    link = "https://docs.openvino.ai/latest/documentation.html"
+    message = '[ INFO ] Starting from 2022.1 release the Model Optimizer generates IR in a format compatible with OpenVINO(TM) API 2.0. ' \
```

Comment:
Since 2022.1 release the Model Optimizer tool produces IR v11 that becomes better aligned with original frameworks in terms of input/output layouts and types. In order to take advantage of this capability, use OpenVINO(TM) API 2.0 for inference. For more information about the API 2.0 please follow: https://docs.openvino.ai. The behavior of inference for IR v11 via the API 1.0 stays unchanged and similar to IR v10.

Reply:
done